.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 180
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - NeverStale
```

Comment:
Minor thing. Is it possible to have spaces? All of our other labels space separate words.

Reply:
Huh I was looking for ages how to correctly put spaces in YAML
Turns out it "just works" I'll do that now :+1: